tests/end-to-end/upgrade/internal/runner/runner.go

```diff
@@ -240,13 +248,16 @@ func (r *TestRunner) wgWait(stopCh <-chan struct{}, wg *sync.WaitGroup) {
 // newLoggerForTask returns new logger which can be used in given task.
 // We need to create new instance of logger otherwise we will start
 // mix logs between each task cause they will share same instance.
-func (r *TestRunner) newLoggerForTask() *logrus.Entry {
-	taskID := r.generateTaskID()
+func (r *TestRunner) newLoggerForTask() (*logrus.Entry, error) {
+	taskID, err := r.generateTaskID()
+	if err != nil {
+		return nil, err
```

Comment:
add error context. e.g `errors.Wrap(err, "while generating ID for task logger")`

Reply:
Done!